Assets/GothicVR/Scenes/Bootstrap.unity

```diff
@@ -865,3 +866,19 @@ MonoBehaviour:
   m_EditorClassIdentifier: 
   dontDestroyOnLoad: 0
   interactionManager: {fileID: 1300502570}
+--- !u!114 &1896514216
+MonoBehaviour:
```

Comment:
When do we actually add the Loading.unity scene? Is it done after Phoenix is bootstrapped? If so, we could think of having this Manager and its materials "not" attached to Bootstrap, but to its own Loading.unity scene. And load the textures once the Loading.unity scene is loaded.